ts/input/tex/base/BaseMethods.ts

```diff
@@ -1961,6 +1961,8 @@ const BaseMethods: { [key: string]: ParseMethod } = {
     if (style === 'S') {
       // @test Subarray, Small Matrix
       array.arraydef['scriptlevel'] = 1;
+    } else {
+      array.arraydef['scriptlevel'] = 0;
```

Comment:
Could be done with a ternary expression of the form:
```suggestion
      array.arraydef['scriptlevel'] = style === 'S' ? 1 : 0;
```
However, this way we get two test paths for completeness.